BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java

```diff
@@ -134,4 +137,22 @@ public TaskState complete(long taskId){
 		return ts;
 	}
 
+
+    public Set<Task> getStartingTask(Date start, String name) {
+		Set<Task> tasks = taskRepo.getSpecificDayTasks(start,name);
+		Set<Task> result = new HashSet<Task>();
+		String pattern = "yyyy-MM-dd";
+		SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
+		String stDate = simpleDateFormat.format(start);
+		if(tasks != null && !tasks.isEmpty()){
+			for(Task task : tasks){
+				SimpleDateFormat simpleD = new SimpleDateFormat(pattern);
```

Comment:
Since the two objects (`simpleD` and `simpleDateFormat`) are exactly same, we can reuse `simpleDateFormat` here instead of creating a new instance.